src/main/target/SITL/dyad/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2016 rxi
```

Comment:
I assume these files are a verbatim copy of 'Dyad.c'? If so, they should be moved to `lib/main/dyad`.